### DISCUSSION TITLE
Verbrauchswerte im Sunny Portal negativ

### DISCUSSION THREAD

**Discussion by starwarsfan** [camueller/SmartApplianceEnabler#560] [Installation und Betrieb]
Hallo miteinander,

nachdem ich eine ganze Weile herumexperimentiert habe, um meine Geräte mit MQTT-Zählerwerten in den SAE resp. in das Sunny Portal (SP) einzubinden, bin ich erst durch [diesen Hinweis](https://github.com/camueller/SmartApplianceEnabler/discussions/531#discussioncomment-7282157) zu einer korrekten Konfiguration gekommen. Vielen Dank an der Stelle nochmals, der [PR dazu](https://github.com/camueller/SmartApplianceEnabler/pull/558) ist gestellt.

Nachdem aber immer noch immer keine Werte im SP angekommen sind, habe ich die Geräte dort gelöscht. Der SAE lief dabei nahtlos weiter. Ein paar Minuten später hat das SP die Geräte wieder gefunden und ich habe sie wieder hinzugefügt. Es werden nun Verbrauchswerte im SP aufgeführt, nur sind diese negativ!? Die angezeigten Werte passen auch überhaupt nicht zu den tatsächlichen MQTT-Verbrauchswerten. Wo liegt hier der Denkfehler, was mach' ich falsch oder was fehlt noch, um zu korrekten Werten zu kommen?

**Comment by camueller** [comment]
Die Geräte im Sunny Portal zu löschen ist fast nie eine gute Idee. Vor allem, wenn man vorher gar nicht geprüft hat, ob der SAE dem SHM an der SEMP-Schnittstelle die korrekten Daten liefert. Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp2

**Reply by starwarsfan** [reply]
Ok, wie es scheint war's nur ein vorübergehender Effekt. Am nächsten Tag u/o einem Neustart des SAE über Nacht war alles soweit ok und die Werte sind seither plausibel.
